src/OpenTelemetry/CHANGELOG.md

```diff
@@ -44,6 +44,10 @@ Released 2021-Jan-29
   [#1501](https://github.com/open-telemetry/opentelemetry-dotnet/pull/1655)
   for details on Metric release plans.
 * Fix Resource attribute telemetry.sdk.version to have correct file version.
+* Default `Resource` has been updated. No longer linked to Telemetry SDK,
```

Comment:
I think a simple one liner like this is sufficient for changelog: Default resource will now contain service.name instead of telemetrysdk.

Reply:
Fixed. Also the line above it, issue #1501 redirects to PR #1655. is this intended?